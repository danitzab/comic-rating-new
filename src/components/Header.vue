<template>
  <header
    class="header p-4 font-rock-salt">
    Comic Rating
  </header>
</template>
