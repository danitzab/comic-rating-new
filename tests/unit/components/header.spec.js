import { shallowMount } from '@vue/test-utils';
import Header from '@/components/Header.vue';

describe('tests in the Header components', () => {
  test('should render the component correctly', () => {
    const wrapper = shallowMount(Header);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
