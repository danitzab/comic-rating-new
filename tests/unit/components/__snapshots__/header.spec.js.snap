// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in the Header components should render the component correctly 1`] = `
<header class="header p-4 font-rock-salt">
  Comic Rating
</header>
`;
